<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理工具</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>授权工具</string>
    </attribute>
    <widget class="QTextBrowser" name="text_auth">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>671</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用户编码</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_limit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_user">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>671</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>结果</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>授权日期</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>授权</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="text_result">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>671</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>邀请工具</string>
    </attribute>
    <widget class="QPushButton" name="pbtn_invite">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>邀请码</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_phone">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>8</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="text_invite">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="table_inviter">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>302</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtn_allInviter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查看</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_invitercount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>授权数据</string>
    </attribute>
    <widget class="QDateEdit" name="date_authlog_start">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_authlog_end">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtn_authlog_refresh">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查看</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_authlog">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>752</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
